awx/main/dispatch/worker/callback.py

```diff
@@ -64,6 +62,7 @@ class CallbackBrokerWorker(BaseWorker):
     """
 
     MAX_RETRIES = 2
+    INDIVIDUAL_EVENT_RETRIES = 3
```

Comment:
do we want to mirror the 5 attempts or we okay with lowering this?

Reply:
It's not that it was trying 5 times before, it would allow 5 errors (from separate events) before giving up saving the working buffer. If it hit over 5 errors, then it would retry the whole batch next time, but if it hit fewer than 5 errors, it would give up on those events right away. So call this a geometric average of 1 and infinity. 3 sounds good. This is giving 3 retries _after we check that postgres connection is working_ so that seems generous to me.

Given the reduced tolerance I'm putting in for the outer loop (adding `self.buffer = {}`), I believe a relevant argument would be to increase `MAX_RETRIES`.

Reply:
👍